src/toil_lib/tools/QC.py

```diff
@@ -27,3 +27,33 @@ def run_fastqc(job, r1_id, r2_id):
     output_files = [os.path.join(work_dir, x) for x in output_names]
     tarball_files(tar_name='fastqc.tar.gz', file_paths=output_files, output_dir=work_dir)
     return job.fileStore.writeGlobalFile(os.path.join(work_dir, 'fastqc.tar.gz'))
+
+
+def run_bam_qc(job, aligned_bam_id, uuid=None):
```

Comment:
Keep in mind that spreading a feature over to repos will make changes to that feature harder. If it is unlikely to be shared by other projects, there is no need to pay that expense. We can always promote it to lib if we feel its necessary.